spec/features/admin/merchants/create_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
I suggest that the name of the file be named new_spec.rb and the erb file name should be new.html.erb